interceptors/2.0/_index.md

```diff
@@ -9,17 +9,17 @@ and other managed classes.
 
 * [Jakarta Interceptors 2.0 Release Record](https://projects.eclipse.org/projects/ee4j.interceptors/releases/2.0.0)
   * [Jakarta EE Platform 9 Release Plan](https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee9/JakartaEE9ReleasePlan)
-* [Jakarta Interceptors 2.0 Specification Document](./interceptors-spec-2.0-SNAPSHOT.pdf) (PDF)
-* [Jakarta Interceptors 2.0 Specification Document](./interceptors-spec-2.0-SNAPSHOT.html) (HTML)
+* [Jakarta Interceptors 2.0 Specification Document](./interceptors-spec-2.0.pdf) (PDF)
```

Comment:
shouldn't the file be named jakarta-....?

Reply:
I think not, the other specs don't have that either:

* https://jakarta.ee/specifications/servlet/5.0/servlet-spec-5.0-SNAPSHOT.html
* https://jakarta.ee/specifications/authentication/2.0/authentication-spec-2.0-SNAPSHOT.html
* https://jakarta.ee/specifications/authorization/2.0/authorization-spec-2.0-SNAPSHOT.html

etc